_posts/en/newsletters/2022-08-10-newsletter.md

```diff
@@ -0,0 +1,184 @@
+  Other node software implements similar policies.
+
+    Lowering the default minimum feerate has been discussed in the past
+    (see [Newsletter #3][news3 min]) but [hasn't been merged][bitcoin
+    core #13922] into Bitcoin Core.  The topic saw renewed
+    [discussion][chauhan min] in the past couple weeks:
+
+    - *Individual change effectiveness:* it was [debated][todd min] by
+      [several][vjudeu min] people how effective it was for individual
+      node operators to change their policies.
+
+    - *Past failures:* it was [mentioned][harding min] that the previous
+      attempt to lower the default feerate was hampered by the lower
+      rate also reducing the cost of several minor denial-of-service
+      (DoS) attacks.
+
+    - *Alternative relay criteria:* it was [suggested][todd min2] that
+      transactions violating certain default criteria (such as the
+      default minimum feerate) could instead fulfill some separate
+      criteria that make DoS attacks costly---for example, if a modest amount
+      of hashcash-style proof of work committed to the transaction to
+      relay.
+
+    The discussion did not reach a clear conclusion as of this writing.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Decouple validation cache initialization from ArgsManager][review club 25527]
+is a PR by Carl Dong that separates node configuration logic from the
+initialization of signature and script caches.
+It is part of the [libbitcoinkernel project][].
+
+
+{% include functions/details-list.md
+  q0="In your own words, what does the `ArgsManager` do?
+Why or why not should `ArgsManager` belong in `src/kernel` versus `src/node`?"
+  a0="It's a global data structure for handling configuration options
+(`.bitcoin/bitcoin.conf` and command line arguments).
+It lets users customise the configuration of their nodes.
+It's undesirable for the consensus engine (`src/kernel`) to access `ArgsManager` for
+any reason, because it could, in principle, allow per-node configuration options
+to modify the behavior of the consensus engine. This could lead to
+a loss of consensus between nodes. Also, when the [libbitcoinkernel project][] is
+complete, it will be possible to build a simple node that's barely more than
+the consensus engine, and might not even have this global data structure.
+Reducing dependency on globals is always an improvement."
+  a0link="https://bitcoincore.reviews/25527#l-35"
+
+  q1="In your own words, what are the validation caches? Why would they belong in
+`src/kernel` versus `src/node`?"
+  a1="When a new block arrives, the most computationally expensive part of validating
+it is validating its transactions; this is part of the consensus engine.
```

Comment:
More specifically than transactions (which is most of the data anyway), script and signature verification is expensive, which is why we cache it.